authors/apps/articles/views.py

```diff
@@ -1,4 +1,5 @@
 """ Views for django Articles. """
+import os
```

Comment:
Is there anywhere you are using the `os` you have imported?

Reply:
I was using in the previous version. I forgot to pluck it out. Thanks for the reminder.